### PR TITLE
Adds export examples for file formats CSV & Excel

### PR DESCRIPTION
**Pull request by athityakumar**
- [x] Download as CSV
- [x] Download as Excel
- [x] Download as JSON
- [x] Download as RDS
- [ ] Download as Avro

I'll add the download as JSON in another PR, once https://github.com/athityakumar/daru-io/pull/40 is merged.

### CODE REVIEW

**Comment by Shekharrajak** [comment]
It will be really helpful if you can show the changes using gif or screen record something like [this](https://github.com/Shekharrajak/daru_examples_io_view_rails/pull/4#issuecomment-319760525).

File: app/helpers/sciruby_github_helper.rb [hunk 1]
```diff
@@ -1,2 +1,26 @@
 module ScirubyGithubHelper
+  def export(dataframe)
+    respond_to do |format|
+      format.html
+
```
Comment (athityakumar) [inline app/helpers/sciruby_github_helper.rb:5]: @Shekharrajak - Could this `export` method be added to daru-view as a method when daru-io is also added? (optional dependency)

**Comment by athityakumar** [comment]
Sorry, couldn't make a gif. But, here are a few screenshots. Here are screenshots of the downloaded files on clicking the respective links (using daru-io exporters). 😄 

- CSV Exporter
image

- Excel Exporter
image

- JSON Exporter
image

- RDS Exporter
image

**Comment by athityakumar** [comment]
I don't think 'download as avro' should be added, as each dataframe has to be given a different schema. Rest of the Exporters have been provided with 'download as' buttons.

**Comment by athityakumar** [comment]
@Shekharrajak - Any changes required in this PR?

**Comment by Shekharrajak** [comment]
Feel free to merge. @athityakumar

**Pull request merged**